src/about.js

```diff
@@ -1,11 +1,12 @@
 const About = () => {
-// hovering event
-// keyboard event
-// Scroll event
-// click event
+    // hovering event
+    // click event
+    // keyboard event
+    // scroll event
```

Comment:
You may begin to remove lines that were used for instructional purposes and have been commented out. This makes your code cleaner to read. 

Nevertheless, you may add comments to your code only if it helps the reader understand what is going on.

Reply:
ok sir